.devcontainer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ghcr.io/eclipse-velocitas/devcontainer-base-images/python:v0.1.3
+
+# Force dapr to use localhost traffic
+ENV DAPR_HOST_IP="127.0.0.1"
+# Add daprd to the path for the VS Code Dapr extension.
+ENV PATH=$PATH:/home/vscode/.dapr/bin
+
+ENV DOCKER_BUILDKIT=1
+
+COPY scripts/*.sh /tmp/scripts/
+RUN find /tmp/scripts/ -type f -iname "*.sh" -exec chmod +x {} \;
+RUN /bin/bash /tmp/scripts/container-set.sh
```

Comment:
You need to add user before.
See: https://github.com/eclipse-velocitas/devcontainer-base-images/blob/improvement/devcontainer/.devcontainer/Dockerfile
```
[dev_container_auto_added_stage_label 4/5] RUN /bin/bash /tmp/scripts/contain
er-set.sh:
#0 0.626 /tmp/scripts/container-set.sh: line 15: $'\r': command not found
#0 0.626 #######################################################
#0 0.626 ### Checking container creation                     ###
#0 0.626 #######################################################
'0 0.630 useradd: invalid option -- '
#0 0.630 Usage: useradd [options] LOGIN
#0 0.630        useradd -D
#0 0.630        useradd -D [options]
#0 0.630
#0 0.630 Options:
#0 0.630       --badnames                do not check for bad names
#0 0.630   -b, --base-dir BASE_DIR       base directory for the home directory o
f the
#0 0.630                                 new account
#0 0.630       --btrfs-subvolume-home    use BTRFS subvolume for home directory
#0 0.630   -c, --comment COMMENT         GECOS field of the new account
#0 0.630   -d, --home-dir HOME_DIR       home directory of the new account
#0 0.630   -D, --defaults                print or change default useradd configu
ration
#0 0.630   -e, --expiredate EXPIRE_DATE  expiration date of the new account
#0 0.630   -f, --inactive INACTIVE       password inactivity period of the new a
ccount
#0 0.630   -g, --gid GROUP               name or ID of the primary group of the
new
#0 0.630                                 account
#0 0.630   -G, --groups GROUPS           list of supplementary groups of the new
#0 0.630                                 account
#0 0.630   -h, --help                    display this help message and exit
#0 0.630   -k, --skel SKEL_DIR           use this alternative skeleton directory
[2023-07-06T12:30:15.763Z] 
#0 0.630   -K, --key KEY=VALUE           override /etc/login.defs defaults
#0 0.630   -l, --no-log-init             do not add the user to the lastlog and
#0 0.630                                 faillog databases
#0 0.630   -m, --create-home             create the user's home directory
#0 0.630   -M, --no-create-home          do not create the user's home directory
#0 0.630   -N, --no-user-group           do not create a group with the same nam
e as
#0 0.630                                 the user
#0 0.630   -o, --non-unique              allow to create users with duplicate
#0 0.630                                 (non-unique) UID
#0 0.630   -p, --password PASSWORD       encrypted password of the new account
#0 0.630   -r, --system                  create a system account
#0 0.630   -R, --root CHROOT_DIR         directory to chroot into
#0 0.630   -P, --prefix PREFIX_DIR       prefix directory where are located the
/etc/* files
#0 0.630   -s, --shell SHELL             login shell of the new account
#0 0.630   -u, --uid UID                 user ID of the new account
#0 0.630   -U, --user-group              create a group with the same name as th
e user
#0 0.630   -Z, --selinux-user SEUSER     use a specific SEUSER for the SELinux u
ser mapping
#0 0.630       --extrausers              Use the extra users database
#0 0.630
' does not exist: user 'vscode
```

Reply:
why dont we need this in python-template?